Sources/TokamakCore/Modifiers/zIndex.swift

```diff
@@ -0,0 +1,31 @@
+// Copyright 2020 Tokamak contributors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+public struct ZIndexModifier : ViewModifier {
```

Comment:
You've used underscored `_ViewModifier_Content` above, I like that idea in principle when applied to symbols that are necessarily public, but aren't re-exported by the `TokamakDOM` module. I don't think there's a better way to indicate that something should stay "private" within the package as a whole, and most probably `TokamakCore` will stay an "internal" module, unless we see a strong use case for 3rd-party renderers (for other platforms maybe?).

```suggestion
public struct _ZIndexModifier : ViewModifier {
```